packages/client/utils/GitHubManager.ts

```diff
@@ -1,14 +1,13 @@
 import {DocumentNode} from 'graphql-typed'
 import {ICreateIssueInput} from '../types/githubGraphql'
-import {IGraphQLResponseError} from '../types/graphql'
 import createIssue from './githubQueries/createIssue.graphql'
 import getProfile from './githubQueries/getProfile.graphql'
 import getRepoInfo from './githubQueries/getRepoInfo.graphql'
 import getRepos from './githubQueries/getRepos.graphql'
 
 export interface GQLResponse<TData> {
   data?: TData
-  errors?: IGraphQLResponseError[]
```

Comment:
`IGraphQLResponseError` isn't being exported from Relay so I've used an `any` here :(

Reply:
this is fine, i've been needing to move our github manager server side anyways